keras_nlp/metrics/bleu.py

```diff
@@ -93,7 +93,19 @@ class Bleu(keras.metrics.Metric):
                not specified, it defaults to tf.float32.
         name: string. Name of the metric instance.
         **kwargs: Other keyword arguments.
-
+        
+    Examples:
+            ```python
+            bleu = keras_nlp.metrics.Bleu(max_order=4)
+            # reference sentence
+            ref_sentence = "the quick brown fox jumps over the lazy dog"
+            # predicted sentence
+            pred_sentence = "the quick brown fox jumps over the box"
+            # compute BLEU score
+            score = bleu([ref_sentence], [pred_sentence])
+            print("BLEU score:", score)
+            ```
```

Comment:
Oh, shoot! I must have forgotten to add examples. Instead of fenced doc-strings, can we have examples this way: https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/metrics/rouge_l.py#L46-L113?

Reply:
yeah sure!